### PR TITLE
travis: install opencpu-server

### PR DESCRIPTION
**Pull request by kortschak**
This change is necessary for running local tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 language: go
 go:
     - 1.6.x
     - 1.7.x
     - 1.8.x
     - tip
+
 install:
     - go get -v golang.org/x/tools/cmd/cover
+    - sudo add-apt-repository -y ppa:opencpu/opencpu-1.6
```
Comment (jeroen) [inline .travis.yml:14]: You should really use the new `opencu-2.0` repo

**Comment by kortschak** [comment]
Yes, but travis does not support xenial and afaics trusty does not have opencpu-2.0.

File: .travis.yml (hunk 1 above)
Comment (kortschak) [reply .travis.yml:14]: It looks like 2.0 breaks the API for starting the server. This is unfortunate. Can I clarify that the change from `opencpu$start(port)` to `ocpu_start_server(port = 5656, root = "/ocpu", workers = 2,
preload = NULL, on_startup = NULL, no_cache = FALSE)` happened in 2.0.0?

Comment (jeroen) [reply .travis.yml:14]: Yes I'm sorry about that, the single user server API has indeed changed in v2.0. Does the new API not work for you/

Comment (kortschak) [reply .travis.yml:14]: Yes, the server [start code for the local session](https://github.com/kortschak/arrgh/blob/450fbc945b18ed6f159a6e22755a3365092454aa/arrgh.go#L75) depends on the `opencpu$start` function. I have written a conditional to sniff the version and decide which start function to call.

Comment (kortschak) [reply .travis.yml:14]: Oh. You've removed the stop server function as well. Is there another undocumented way to stop the server that is not a signal (sending signals makes session control way more complicated)?

**Pull request merged**